doc/api/errors.md

```diff
@@ -1231,6 +1231,14 @@ range.
 
 Used by the `REPL` module when it cannot parse data from the REPL history file.
 
+<a id="ERR_REPL_STARTSIGINTWATCHDOG_FAILED"></a>
+### ERR_REPL_STARTSIGINTWATCHDOG_FAILED
+
+The `SigintWatchdog` is an internal utility used by the `repl` module to
```

Comment:
Is it too much detail? AFAICT this is triggered by a failed `sigmask`, may be we don't have to mention the watch dog thing, simply saying that node is unable to monitor `SIGINT` is enough? (`ERR_REPL_SIGINTWATCH_FAILED`)

Reply:
EDIT: paste error, `s/sigmask/pthread_create/`

Reply:
I agree, both the error code & the docs on it do have too much detail – `ERR_CANNOT_WATCH_SIGINT` or so should be fine.